server/utils/middleware.js

```diff
@@ -29,21 +29,17 @@ export function ifNoUser401(req, res, next) {
   return res.status(401).end();
 }
 
-export function flashIfNotVerified(req, res, next) {
-  return next();
-  /*
-  // disabled until authorized required bug is fixed
-  const user = req.user;
+export function ifNoEmailVerifiedRedirectToSettings(req, res, next) {
```

Comment:
We can shorten this to just `ifNotVerifiedRedirectToSettings` as we will be using the verified field more often now we don't need the extra `email` qualifier.